Readme.md

```diff
@@ -165,8 +165,54 @@ It is not a problem to render async components with `react-snap`, the tricky par
 
 Usually _code splitting_ library provides an API to handle it during SSR, but as long as "real" SSR is not used in react-snap - the issue surfaces, and there is no simple way to fix it.
 
-1. use `loadable-components` 2.2.3 (current is >5)
-The old version of `loadable-components` can solve this issue for a "snapshot" setup:
+1. Use [react-prerendered-component](https://github.com/theKashey/react-prerendered-component)
+   This library could _keep_ the pre-rendered HTML until codesplitted part is ready.
```

Comment:
Use [react-prerendered-component](https://github.com/theKashey/react-prerendered-component). This library holds onto the prerendered HTML until the dynamically imported code is ready.